spring-cloud-schema-registry-server/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM java:8-alpine
+
+ARG JAR_FILE
+
+ADD target/${JAR_FILE} spring-cloud-skipper-server.jar
+
+ENTRYPOINT exec java -Djava.security.egd=file:/dev/./urandom -jar /spring-cloud-skipper-server.jar
```

Comment:
Again replace the `spring-cloud-skipper-server.jar` by `spring-cloud-schema-registry.jar` instead.